liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojo.java

```diff
@@ -108,6 +109,9 @@ private void doDeploy() throws IOException, MojoExecutionException, TransformerE
             getLog().warn(messages.getString("warn.install.app.add.configuration"));
             applicationXml.writeApplicationXmlDocument(serverDirectory);
         }
+        
+        // create applicationMonitor configuration in configDropins/defaults
+        appMonXml.writeAppMonitorConfigXmlDocument(serverDirectory, appMonitorTrigger);
```

Comment:
I can see the value in grouping this together with writing the  install_apps_...xml.  We want to limit the number of places we're generating config from.

That said, I wonder if the copyConfigFiles() routine is the better place.    

Within the dev mode use case it's hard to come up with a distinction between the two.

I guess say I was using the goals outside of dev mode.. I have done a deploy earlier and am now doing compiles and start and stop of the server. 

Do we want the plugin configuration at the time of deploy to still basically take effect now, later?  Or if I run without this new parameter set, should I revert back to the default state (and have LMP delete the configDropin)?

I'm just going to pose the question for now and come back and think more as well.